### ISSUE TITLE
(php) Impossible to install two versions of PHP

### ISSUE THREAD

**Issue by arturolinares** [ferventcoder/chocolatey-packages#115]
If I want to install two versions of PHP using the side-by-side switch (`install -m`), the last version installed will replace the first one because both use the same setup dir (\tools\php).

```
> choco install php --version 5.6.17
> php --version
PHP 5.6.17 (cli) (built: Jan  6 2016 13:28:18)
...
> choco install php --version 7.0.2
> php --version
PHP 7.0.2 (cli) 
...

```

If the setup dir includes the version when using `-m`, then switching between the two would be then a change in the PATH:

To use php 5.6.17
`PATH=%PATH%;\tools\php5.6.17`

To use php 7
`PATH=%PATH%;\tools\php7.0.2`

**Comment by ferventcoder** [comment]
Is PHP like Ruby in that it can be installed multiple times?

**Comment by laurin1** [comment]
Yes.

**Comment by ferventcoder** [comment]
@majkinetor created https://github.com/chocolatey/chocolatey-coreteampackages/issues/425 to address this.

**Comment by ferventcoder** [comment]
We'll keep it open here until it is implemented there.

**Comment by majkinetor** [comment]
@ferventcoder, can I recognize the -m switch from within chocolateyInstaller ?

**Comment by ferventcoder** [comment]
Nada - the trick is usually to explicitly set the install directory to `php###` where `#` is the accepted versioning convention that is most widely used - for instance it could be `5.4` or `5.4.x` and then `5.5`. I am not typically using php so I don't know what that should look like. 

In Ruby it is:
- Ruby1.8.7
- Ruby1.9.3
- Ruby2.0.0
- Ruby2.1
- Ruby2.3

Or something like that. Away from my computer at the moment, the versioning changed after 2.0.0 to be more SemVer compliant (at least in numbering).

**Comment by majkinetor** [comment]
Rob, that is not the problem. I already did the same thing for python. The problem is how to handle existing users on update. My plan was to still install to the old location if you don't care for multiple versions. Since that is not possible installer code would have to check for older installation.

**Comment by ferventcoder** [comment]
usually folks use `-m` (or side by side installs) and then they pin those installs so they don't upgrade. There is a better story for upgrades in that kind of set up coming later.

**Comment by photodude** [comment]
I'm assuming this issue can be closed now that @AdmiringWorm has push #548 with the function for this

**Comment by ferventcoder** [comment]
Likely.
